src/courtroom.cpp

```diff
@@ -4441,11 +4447,9 @@ void Courtroom::set_iniswap_dropdown()
   }
   QStringList iniswaps =
       ao_app->get_list_file(ao_app->get_character_path(char_list.at(m_cid).name, "iniswaps.ini")) +
+      ao_app->get_list_file(VPath("iniswaps.ini")) +
       ao_app->get_list_file(ao_app->get_base_path() + "iniswaps.ini");
```

Comment:
This causes duplicate entries when editing the iniswaps.ini
[bugs.webm](https://user-images.githubusercontent.com/26681464/181112303-8a31c5e6-f169-477e-ba78-84264e535d25.webm)
.